src/init/init.ts

```diff
@@ -156,12 +156,11 @@ function getDisplayName(generatorName: string) {
   // -------------------------------------------------------------------
   // (generator-)?           | Grp 1; Match "generator-"; Optional
   // (polymer-init)?         | Grp 2; Match "polymer-init-"; Optional
-  // (polymer-starter-kit-)? | Grp 3; Match "polymer-starter-kit-"; Optional
```

Comment:
I think it would be easier to replace this commit with `git revert e063a60` to undo the [commit that added the `polymer-starter-kit-` prefix](https://github.com/Polymer/polymer-cli/pull/472/commits/e063a604dc9306eff0c6fdca354bfa4fd5e95258).